pose-detection/src/blazepose_mediapipe/README.md

```diff
@@ -140,3 +141,16 @@ There is a difference of how things are loaded between the two runtimes. For the
 | Lite model | 10.41MB | 1.91s |
 | Full model | 13.8MB | 1.91s |
 | Heavy model | 34.7MB | 4.82s |
+
+## Model Accuracy
+This table compares MediaPipe BlazePose accuracy against other state-of-the-art
```

Comment:
Please add the description about verticals and rename Accuracy to Quality (Accuracy is another well known metric, so let's not confuse readers). 

Pose Estimation Quality
To evaluate the quality of our models against other well-performing publicly available solutions, we use three different validation datasets, representing different verticals: Yoga, Dance and HIIT. Each image contains only a single person located 2-4 meters from the camera. To be consistent with other solutions, we perform evaluation only for 17 keypoints from COCO topology.

Reply:
Link to topology:  https://cocodataset.org/#keypoints-2020